src/components/partials/profile/ProfileHeader.vue

```diff
@@ -234,7 +234,13 @@ export default class ProfileHeader extends Vue {
     }
 
     sendToken() {
-        this.sendTokenModalOpen = true;
+        if (this.web3Accounts.length === 0) {
+            this.toast.error(
+                "Add wallet connection with Polkadot.js extension to use this feature"
```

Comment:
@sosan-neikbeen Please use the following text:
```
To use this feature, please attach a wallet connection with polkadot{.js} extension. Click here for more help.
```

Is it possible to have a link in the toast? 

Would be nice to redirect to: https://parallelfi.gitbook.io/parallel-finance/polkadot-network/parallel-product-guides/a-guide-to-testnet/polkadot-.js-connect-wallet